markdown/developers/tutorials/unofficial_pipelines.md

```diff
@@ -0,0 +1,93 @@
+
+```markdown
+This pipeline uses code and infrastructure developed and maintained by the [nf-core](https://nf-co.re) initiative, and reused here under the [MIT license](https://github.com/nf-core/tools/blob/master/LICENSE).
+
+> The nf-core framework for community-curated bioinformatics pipelines.
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
+>
+> Nat Biotechnol. 2020 Feb 13. doi: 10.1038/s41587-020-0439-x.
+>
+In addition, references of tools and data used in this pipeline are as follows:
+```
+
+## Removing nf-core only branding
+
+> Most of the following suggestions refer to when you have started your pipeline using the nf-core template.
+
+## General
+
+- Replace in all places reference to the pipeline name that includes the `nf-core/` suffix.
+
+### `README`
+
+To ensure users do not get confused, we suggest removing the following components of the main pipeline README
+
+1. Slack badge for help messages.
+2. Replace all references of the [https://nf-co.re](https://nf-co.re) website to the location of your own documentation
+3. Remove reference to the nf-core Slack workspace under `Contributions and Support
+4. Replace the `Citations` section with the following markdown:
+
+    ```markdown
+    This pipeline uses code and infrastructure developed and maintained by the [nf-core](https://nf-co.re) initative, and reused here under the [MIT license](https://github.com/nf-core/tools/blob/master/LICENSE).
+
+    > The nf-core framework for community-curated bioinformatics pipelines.
+    >
+    > Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
+    >
+    > Nat Biotechnol. 2020 Feb 13. doi: 10.1038/s41587-020-0439-x.
+    >
+    In addition, references of tools and data used in this pipeline are as follows:
+    ```
+
+Optional:
+
+1. Reference to nf-core/configs in `Quick Start`
+     - Note in many cases it is still reasonable to use these configs for non-nf-core pipelines as they typically only modify generic Nextflow options
+
+### `docs/`
+
+1. Replace the nf-core logo under `docs/images` with your own one.
+     - You will also have to change the filename/path at the top the main repository README
+2. `docs/usage.md`: remove reference to documentation being viewable on the [nf-co.re](https://nf-co.re) website
+3. `docs/usage.md`: remove reference to the nf-core Slack workspace
+
+Optional:
+
+1. Reference to nf-core/configs in `-profile`
+     - Note in many cases it is still reasonable to use these configs for non-nf-core pipelines as they typically only modify generic Nextflow options
+
+### `assets/`
+
+1. `*email_template.*` - remove references to `nfcorepipelinelogo`
```

Comment:
Though if swapped for a different image then maybe this doesn't matter so much..?

Reply:
I'm not actually sure what the `cid` refers to, so thought it safer to remove :grimacing: 

https://github.com/nf-core/tools/blob/7721f3742fc30a9e6e46d879404bd1366345092d/nf_core/pipeline-template/assets/email_template.html#L13

Reply:
It's an arbitrary identifier.. It's needed for the attachment / image but it doesn't matter what it is (not visible anywhere unless you look hard when the email is sent).

Reply:
So if people remove it then the email logo attachment will break. If they replace it with another string it should be fine. If they just leave it alone it shouldn't really matter too.